using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -614,32 +614,110 @@ public static void setAdvertisingIdentifier(final String advertisingIdentifier);
 
 #### **Example**
 
+{% hint style="warning" %}
+
+This is just an implementation example, for more details about advertising identifier and how you should handle it correctly in your mobile application, please refer to [Google Play Services documentation about Advertising ID](http://www.androiddocs.com/google/play-services/id.html).
```

Comment:
Please reword so that the text reads as follows:

"This is just an implementation example. For more information about advertising identifiers and how to handle them correctly in your mobile application, see <link>."